src/libs/ReportUtils.ts

```diff
@@ -4249,10 +4236,13 @@ function buildOptimisticTaskCommentReportAction(
 
     // These parameters are not saved on the reportAction, but are used to display the task in the UI
     // Added when we fetch the reportActions on a report
-    reportAction.reportAction.originalMessage = {
-        html: ReportActionsUtils.getReportActionHtml(reportAction.reportAction),
-        taskReportID: ReportActionsUtils.getReportActionMessage(reportAction.reportAction)?.taskReportID,
-        whisperedTo: [],
+    reportAction.reportAction = {
```

Comment:
`reportAction.originalMessage` throws a lint error. We use originalMessage in building all optimistic report actions in this file. So, just refactored to remove that error.

Reply:
@c3024 We have a PR https://github.com/Expensify/App/pull/49325 addressing the linting of the src/libs/ReportUtils.ts file. What do you think about waiting for the other PR before proceeding?

Reply:
Commented on that PR. Wish I checked that before.

Reply:
> Wish I checked that before

That happens sometimes 😃